### ISSUE TITLE
shorter doesn't short wikipedia links

### ISSUE THREAD

**Issue by ryanthara** [hzlzh/Alfred-Workflows#16]
Hello. First, I want to say thank you for sharing your workflows.

I try to short the following link: https://de.wikipedia.org/wiki/Zurück_in_die_Zukunft#Der_Fluxkompensator

shorturl shows as result the search window with google, amazon and wikipedia. 

<img width="541" alt="shorturl" src="https://cloud.githubusercontent.com/assets/7117072/15038653/dc47e63c-12a5-11e6-9b3f-055b0da1070c.png">

Is there a solution to short this urls?

**Comment by hzlzh** [comment]
Try to use `%23` instead of `#` like this
 https://de.wikipedia.org/wiki/Zurück_in_die_Zukunft%23Der_Fluxkompensator

**Comment by ryanthara** [comment]
Good idea. I tested it and it still works in safari, but the `#` sign (anchor on a website) comes from the wikipedia website and is in all web browsers the same. My coding experience with python is nearly `0` - so would it be possible to do this with a regex for wikipedia (and maybe other) websites?

**Comment by hzlzh** [comment]
Yes, I will fix it since Alfred v3 is coming soon.
